### DISCUSSION TITLE
Realtime Listeners on PostgreSQL Views

### DISCUSSION THREAD

**Discussion by WilliamScott-AdventGX** [supabase/supabase#7849] [Questions] [answered]
Does Supabase have any support to enable realtime listening on a SQL view?

This may not be possible but I am hoping this is some obscure feature of Supabase.

If not, has anyone come up with viable solutions to such a problem?

Thanks!

**Comment by rustamd** [comment, marked as answer]
Check this [discussion](https://github.com/supabase/supabase/discussions/5152) out, might be able to make it work [using this approach](https://github.com/supabase/supabase/discussions/5152#discussioncomment-2091589).

**Reply by WilliamScott-AdventGX** [reply]
I think Osaxma might have a working [solution](https://github.com/supabase/supabase/discussions/5152#discussioncomment-2091589). I will take a poke at it and see.

I appreciate the quick reply and great discussion link.
